<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zhuanye</class>
 <widget class="QWidget" name="zhuanye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>专业介绍</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>90</y>
     <width>1141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/image/timg.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/zhuanyejieshao.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="LinkButton_news_zhuye_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>专业介绍</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="LinkButton_news_zhuye">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>主页</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>中国语言文学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哲学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外国语学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>岭南学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>社会学与人类学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>马克思主义学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>法学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>政治与公共事务管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>资讯管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>传播与设计学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>心理学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>艺术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>博雅学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>化学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生命科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地理科学与规划学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环境科学与工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料科学与工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>逸仙学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数据科学与计算机学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电子与信息工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中山医学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>光华口腔医学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公共卫生学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>护理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>药学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>体育部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国语言文学系（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史系（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哲学系（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>旅游学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际关系学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际金融学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际翻译学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学学院（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大气科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地球科学与工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海洋科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海洋工程与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理与天文学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中法核工程与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>化学工程与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>土木工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>微电子科学与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>测绘科学与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>医学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公共卫生学院（深圳）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>药学院（深圳）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生物医学工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电子与通信工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>航空航天学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>智能工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>农学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生态学院</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>院系选择：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>院系所在校区：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;汉仪蝶语体简&quot;;
background-image: url(:/image/heidi.png);</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>专业设置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>院系介绍：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>近三年平均排位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广东</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>河北</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山东</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>辽宁</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>吉林</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>青海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>河南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江苏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖北</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>浙江</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>云南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>福建</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>台湾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四川</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>陕西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>贵州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>安徽</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>重庆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内蒙古</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西藏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宁夏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新疆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>澳门</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>香港</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>580</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>2018：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>580</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>2017：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>580</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>2016：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>该专业在以下省份不招生：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>831</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>831</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>570</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>570</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_6">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>570</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>640</y>
     <width>711</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>LinkButton_news_zhuye</sender>
   <signal>clicked()</signal>
   <receiver>zhuanye</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
